packages/fsengagement/src/EngagementComp.tsx

```diff
@@ -277,10 +316,55 @@ export default function(
       );
     }
 
+
+    onSnapToItem = (index: number): void => {
+      const pageNum = index + 1;
+      this.setState({
+        pageNum
+      });
+    }
+    // tslint:disable-next-line:cyclomatic-complexity
     renderScrollView(): JSX.Element {
-      const { json: { storyGradient } } = this.props;
+      const { json, json: { storyGradient } } = this.props;
```

Comment:
Is this a typo?

Reply:
@bweissbart no, i wanted to have both json and storyGradient as consts, if i just do 
```
const { json: { storyGradient } } = this.props;
```
that doesn't give me json, only storyGradient.

Reply:
Yep, you're right. New syntax for me!